cmd/wellknownoperator.go

```diff
@@ -35,9 +35,15 @@ type KnownOperators []KnownOperator
 var wellKnownOperators KnownOperators
 
 func defaultWellKnownOperators() KnownOperators {
-	return KnownOperators{
+	list := []KnownOperator{
 		{Name: "synadia", URL: "https://www.synadia.com", AccountServerURL: "https://api.synadia.io/jwt/v2/synadia"},
 	}
+	if os.Getenv("NSC_DEV") != "" {
```

Comment:
the wellknown operator can be defined in the environment instead.

Reply:
thanks,  confirmed locally by setting `NSC_LOCAL_OPERATOR=http://localhost:9090/jwt/v1/operator` and it worked 👍